VisualStudio.gitignore

```diff
@@ -183,6 +183,7 @@ AppPackages/
 BundleArtifacts/
 Package.StoreAssociation.xml
 _pkginfo.txt
+.appx
```

Comment:
This should already be covered by the earlier `AppPackages/` rule, as per the StackOverflow answer.

If not, could you elaborate on why that's not the case?